### PR TITLE
adding a new line in steps to check merging with main branch or not

### PR DESCRIPTION
**Pull request by 3zzazakl**
- rename file and get done word after merging with main branch
- adding a new line to steps file

### CODE REVIEW

**Comment by 3zzazakl** [comment]
done

**Review by 3zzazakl — COMMENTED** [review]
done

**Comment by 3zzazakl** [comment]
do

File: zzsteps.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+3- HashNodes ()
+4- 
+***************************************************************
+                    THE_EXAM****09-01-2024*****THE_EXAM
+                                    (Tasks)
+1- finish console task  (6 + 7) + get info about.
+2- Overview (BaseModel Class) to get data from it to handle with console.
+3- 
+
+***************************************************************
```
Comment (azzaz3kl) [inline zzsteps.txt:22]: setting goals 4today

File: zzsteps.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                    THE_EXAM****09-01-2024*****THE_EXAM
+                                    (Errors)
+1-
```
Comment (azzaz3kl) [inline zzsteps.txt:25]: Setting goals 4today

**Review by azzaz3kl — APPROVED** [review]
done

File: zzsteps.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                    THE_EXAM****09-01-2024*****THE_EXAM
+                                (Resources)
+1- finish Obed Blogs (Challenges + Console).    ()
+2- finish Shaza Blogs (Prep + Console + Unit_Testing + collaborative)
+3- HashNodes ()
+4- 
+***************************************************************
+                    THE_EXAM****09-01-2024*****THE_EXAM
+                                    (Tasks)
+1- finish console task  (6 + 7) + get info about.
```
Comment (3zzazakl) [inline zzsteps.txt:18]: done

File: zzsteps.txt (hunk 1 above)
Comment (3zzazakl) [inline zzsteps.txt:22]: done

File: zzsteps.txt (hunk 2 above)
Comment (3zzazakl) [inline zzsteps.txt:25]: done

**Review by 3zzazakl — COMMENTED** [review]
take a break

**Comment by 3zzazakl** [comment]
done

**Pull request merged**